protected/humhub/modules/user/controllers/PeopleController.php

```diff
@@ -8,11 +8,15 @@
 namespace humhub\modules\user\controllers;
 
 use humhub\components\access\ControllerAccess;
+use humhub\components\behaviors\AccessControl;
 use humhub\components\Controller;
+use humhub\modules\space\permissions\SpaceDirectoryAccess;
 use humhub\modules\user\components\PeopleQuery;
+use humhub\modules\user\helpers\AuthHelper;
 use humhub\modules\user\permissions\PeopleAccess;
 use humhub\modules\user\widgets\PeopleCard;use Yii;
```

Comment:
Not sure if it's a mobile issue or an actual issue, but from viewing it it seems that a line break was missed here.